Makefile

```diff
@@ -517,6 +517,10 @@ bazel_loaddatatest: failpoint-enable bazel_ci_prepare
 	bazel $(BAZEL_GLOBAL_CONFIG) test $(BAZEL_CMD_CONFIG) --test_arg=-with-real-tikv --define gotags=deadlock,intest \
 		-- //tests/realtikvtest/loaddatatest/...
 
+bazel_importintotest: failpoint-enable bazel_ci_prepare
```

Comment:
@hawkingrei what change should we make to let CI run it?

Reply:
see https://github.com/PingCAP-QE/ci/pull/2157

Reply:
first, You should create a new empty with-real-tikv test. then you need to enable it in the CI.  finally, we can merge this PR.

Reply:
after discuss @hawkingrei , will create an empty with-real-tikv test `importintotest` in this pr, after enable it in the CI, will move code in `loaddatatest` into `importintotest`